index.md

```diff
@@ -14,10 +14,7 @@ excerpt: "The Belgium Azure User Group focuses on knowledge sharing and networki
 
 ## Upcoming events
 
-**2019-05-06 - //BUILD Live stream** - Join with your peers and watch the //BUILD 2019 live stream together. [Register here!](https://www.azug.be/events/2019/05/06/build-live-stream){: .btn .btn--success}
-{: .notice--info}
-
-**2019-05-07 - Binding your serverless clients & 6 years of Bootstrapping a SaaS on Azure** - Want to learn how you can write your own Azure Function bindings from Jan de Vries? Or hear Maarten Balliauw talk about 6 years of Bootstrapping a SaaS on Azure? Then this event is for you! [Register here!](https://www.azug.be/events/2019/05/07/binding-your-serverless-clients){: .btn .btn--success}
+**2019-12-13 & 14 - CloudBrew 2019** - Two days of Azure, beers and networking! [Register here!](https://www.cloudbrew.be/){: .btn .btn--success}
```

Comment:
Should we add a page to our events section on this as well? If so, I can take this up.

Reply:
Note registrations are not open yet :-) Maybe "more info"?

And yes, page in events section could be nice (can be copied from last year, mostly)

Reply:
Will do in seperate PR!